pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -85,19 +85,15 @@ self: super:
   });
 
   libxcb = super.libxcb.overrideAttrs (attrs: {
-    nativeBuildInputs = attrs.nativeBuildInputs ++ [ python ];
     configureFlags = [ "--enable-xkb" "--enable-xinput" ];
     outputs = [ "out" "dev" "man" "doc" ];
   });
 
-  xcbproto = super.xcbproto.overrideAttrs (attrs: {
-    nativeBuildInputs = attrs.nativeBuildInputs ++ [ python ];
-  });
-
   libX11 = super.libX11.overrideAttrs (attrs: {
     outputs = [ "out" "dev" "man" ];
     configureFlags = attrs.configureFlags or []
       ++ malloc0ReturnsNullCrossFlag;
+    depsBuildBuild = [ buildPackages.stdenv.cc ];
```

Comment:
Not sure about this one @Ericson2314, this is one of the cases I mentioned a while ago where the build fails because there is no `aarch64-unknown-linux-gnu-cpp` and the build calls `cc -E` wrong. 

This should probably be fixed by adding `aarch64-unknown-linux-gnu-cpp` and not by adding native `cpp`. This is also the case for `ar`.
```
$ ls /nix/store/n4lf26a3mqpnfh1m5rccj4pgxamrkjql-aarch64-unknown-linux-gnu-stage-final-gcc-debug-wrapper-7.3.0/bin/
aarch64-unknown-linux-gnu-as   aarch64-unknown-linux-gnu-cc   aarch64-unknown-linux-gnu-gcc  aarch64-unknown-linux-gnu-ld.bfd
aarch64-unknown-linux-gnu-c++  aarch64-unknown-linux-gnu-g++  aarch64-unknown-linux-gnu-ld   aarch64-unknown-linux-gnu-ld.gold
```

Reply:
No cpp? Woah that's weird. I could have sworn their once was.

Reply:
Fwiw, I have similar issue with libXt.

Reply:
I also have seen this problem somewhere else.